### DISCUSSION TITLE
Issues trying to reach localhost:3001/status

### DISCUSSION THREAD

**Discussion by juliomarmin** [Kong/insomnia#7799] [Help]
I have Insomnia 8.6.1 on Windows

I have a local application listening on port 3001, and when I try to reach it with localhost:3001/status , I get in Insomnia 'Not Found'
If I copy the cURL command and try in a shell, I get response. 

I changed the listening port to 3002, and with localhost:3002/status Insomia can get a response!

When listening on 3001, Insomnia works with url [::1]/status

I can't figure out why Insomnia does not work with localhost:3001, but works with other ports.

cURL works fine.  

What's going on?

**Comment by jeremytm** [comment]
Something similar happening for me but on macOS

```
Version: Insomnia 11.2.0
Build date: 6/5/2025
OS: Darwin arm64 24.5.0
Electron: 35.1.5
Node: 22.14.0
Node ABI: 133
V8: 13.4.114.21-electron.0
Architecture: arm64
```
<img width="947" alt="image" src="https://github.com/user-attachments/assets/3b217f2d-e366-4a97-9665-f6c9760b0c3e" />
<img width="369" alt="image" src="https://github.com/user-attachments/assets/518aab5e-92e7-4dd9-beb0-249466561c33" />
